docusaurus/docs/reactnative/common-content/contexts/channel-context/scroll_to_first_unread_threshold.mdx

```diff
@@ -0,0 +1,6 @@
+Minimum number of unread messages a channel should have, to set the initial scroll position to first unread message.
+This value is only used when [initialScrollToFirstUnreadMessage](/reactnative/core-components/channel/#initialscrolltofirstunreadmessage) is set to true.
```

Comment:
maybe link to he file? even if you're on common-content links to files should work as well.